src/markdown-pages/contribute-to-io/build-a-quickstart/create-a-quickstart.mdx

```diff
@@ -0,0 +1,470 @@
+violationTimeLimitSeconds: 259200
+```
+
+## Add a description
+
+Your quickstart is almost ready to go live. But how will others know what your quickstart is for and what it can do?
+
+To help others understand the purpose of your quickstart, add a description. 
+
+In the root directory of your pack, you can find a _config.yml_ file. 
+Open the file in your IDE and edit it as follow:
+
+```yml
+ 
+# Name of the pack (required)
+name: flashdb
+
+# Description of the pack (required)
+description: |
+  Flashdb is the fastest database in the space. 
+  The Flashdb quickstart allows you to get visibility into the performance of flashdb with the help of following features:
+
+  Dashboards:
+  - Database methods: Monitor the number of transactions for each database method
+  - Database size: Monitor the size of database
+  - Average response time: Monitor the average response time for each type of database transaction
+  - Cache hits: Monitor the cache hits
+  - Errors: Monitor the errors occured for each type of database transaction
+  - Database keys: Monitor the keys for database
+ 
+  Alerts:
+  - Cache hit ratio: Alert is triggered when cache hit ratio falls below the set limit.
+  - Read response: Alert is triggered when read responses are slow and exceeds a set limit
+
+summary: |
+  Flashdb is the fastest database in the space. 
+  This quickstart allows you to get visibility into the performance of flashdb.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Design
+icon: icon.jpeg
+logo: logo.png
+
+# Authors of the pack (required)
+authors:
+  - {"enter your name"}
+
+# References to other packs (optional)
+# References are grouped by the type: operating-system, database, webserver, queue, ..
+# The reference under the type is a at least one system, so at least one operating-system, database, ..
+references:
+  - type: database
+    name:
+    - flashdb
+```
+
+Here's how your final _flashdb_ quickstart folder should look like.
+![flashdb quickstart directory](../../../images/contribute-to-io/final-flashdb-quickstart.png)
+
+## Contribute quickstart to GitHub
+
+Your quickstart is now ready to be published. You're going to commit your changes back to GitHub where it will be reviewed by New Relic. 
+Follow the [conventional commit syntax](https://github.com/newrelic/newrelic-observability-packs/blob/alerts-doc-update/CONTRIBUTING.md#using-conventional-commits) for New Relic to commit your changes.
+
+```bash
+git add -A
+git commit -m "feat(flashdb): Added a quickstart for flashdb"
+```
+
+<Callout variant="important">
+
+We, at New Relic, are always happy to receive contributions from the community. However, we are expecting a number of contributions for flashdb following the same lab course. 
+To help us manage your contributions efficiently, we encourage you to label your contribution following this particular lab as **lab** using `git tag` command.
```

Comment:
I think it could be helpful to provide the user with the entire command as a part of the following code block 

`git tag lab`